src/pudl/transform/nrelatb.py

```diff
@@ -514,10 +534,6 @@ def check_technology_specific_parameters(df):
                 "heat_rate_penalty",
             ],
         },
-        {
-            "technology_descriptions": {"OffShoreWind"},
-            "params": ["capex_grid_connection_per_kw"],
-        },
```

Comment:
Why did this get removed?

Reply:
Noted above in the PR description, they expanded the data that this column is reported for:

> Removed `capex_grid_connection_per_kw` from `check_technology_specific_parameters` - this is now a variable for all technology types (see https://atb.nrel.gov/electricity/2024/changes_in_2024 for documentation of this change, which is expected)